desc/compute/_curve.py

```diff
@@ -212,8 +212,11 @@ def _x_FourierPlanarCurve(params, transforms, profiles, data, **kwargs):
     # rotate into place
     Zaxis = jnp.array([0.0, 0.0, 1.0])  # 2D curve in X-Y plane has normal = +Z axis
     axis = cross(Zaxis, normal)
-    angle = jnp.arccos(dot(Zaxis, safenormalize(normal)))
-    A = rotation_matrix(axis=axis, angle=angle)
+    angle = safearccos(dot(Zaxis, safenormalize(normal)))
+    if jnp.allclose(Zaxis + normal, 0.0):  # normal is anti-parallel to +Z axis
+        A = jnp.diag(jnp.array([1.0, -1.0, -1.0]))
+    else:
+        A = rotation_matrix(axis=axis, angle=angle)
```

Comment:
Dario's old solution had this logic in the `rotation_matrix` function. I want to keep that function general for arbitrary rotations, without assuming the edge case of being anti-parallel is relative to the Z-axis. The downside is that now we have to handle this logic in each of these compute functions.